src/VisualStudio/LiveShare/Impl/Client/RemoteLanguageServiceWorkspace.cs

```diff
@@ -39,7 +40,8 @@ namespace Microsoft.VisualStudio.LanguageServices.LiveShare.Client
     /// A Roslyn workspace that contains projects that exist on a remote machine.
     /// </summary>
     [Export(typeof(RemoteLanguageServiceWorkspace))]
-    internal sealed class RemoteLanguageServiceWorkspace : CodeAnalysis.Workspace, IDisposable, IRunningDocumentTableEventListener
+    [Export(typeof(IVsTypeScriptRemoteLanguageServiceWorkspace))]
```

Comment:
could you link the tracking item for the LSP based solution for these features here so we know when we no longer need this?

Reply:
Added to the interface we'd use to import the accessor.